backend/store/etcd/silenced_store.go

```diff
@@ -190,3 +214,23 @@ func (s *Store) arraySilencedEntries(resp *clientv3.GetResponse) ([]*types.Silen
 	}
 	return silencedArray, nil
 }
+
+func (s *Store) arrayTxnSilencedEntries(ctx context.Context, resp *clientv3.TxnResponse) ([]*corev2.Silenced, error) {
+	results := []*corev2.Silenced{}
+	for _, resp := range resp.Responses {
+		for _, kv := range resp.GetResponseRange().Kvs {
+			leaseID := clientv3.LeaseID(kv.Lease)
+			ttl, err := s.client.TimeToLive(ctx, leaseID)
+			if err != nil {
+				return nil, fmt.Errorf("couldn't get silenced entries: %s", err)
+			}
+			var silenced corev2.Silenced
+			if err := unmarshal(kv.Value, &silenced); err != nil {
+				return nil, fmt.Errorf("couldn't get silenced entries: %s", err)
```

Comment:
Another nitpick, we reuse the error statement `couldn't get silenced entries` about 5 times. We _could_ consolidate.

Reply:
Is the suggestion:

```
func silencedErr(err error) error {
    return fmt.Errorf("couldn't get silenced entries: %s", err)
}
```
and use that function when errors are returned?

Reply:
A function, or a constant. Or do nothing at all. Just something I noticed.

Reply:
I've done this in the past, but have come to conclude that the indirection isn't worth it when reading the code.